GPL/EventProbe/Helpers.h

```diff
@@ -22,13 +22,30 @@
 #define EBPF_EVENTPROBE_HELPERS_H
 
 #include "EbpfEventProto.h"
-#include "FileEvents.h"
 
 #if BPF_DEBUG_TRACE == 0
 #undef bpf_printk
 #define bpf_printk(fmt, ...)
 #endif
 
+#define DECL_RELO_FUNC_ARGUMENT(func_name, arg_name)                                               \
+    const volatile int arg__##func_name##__##arg_name##__ = 0;
+#define RELO_FENTRY_ARG_READ(type, func_name, arg_name)                                            \
+    ({                                                                                             \
+        type _ret;                                                                                 \
+        bpf_core_read(&_ret, sizeof(unsigned long long),                                           \
+                      ctx + arg__##func_name##__##arg_name##__);                                   \
+        _ret;                                                                                      \
+    })
+
+#define DECL_RELO_FUNC_RET(func_name) const volatile int ret__##func_name##__ = 0;
+#define RELO_FENTRY_RET_READ(type, func_name)                                                      \
+    ({                                                                                             \
+        type _ret;                                                                                 \
+        bpf_core_read(&_ret, sizeof(unsigned long long), ctx + ret__##func_name##__);              \
```

Comment:
```suggestion
        bpf_core_read(&_ret, sizeof(_ret), ctx + ret__##func_name##__);              \
```
right? when `_ret` is an `int` we would be reading 64bits (`long long`) and storing in 32bits (`int`) right?

Reply:
Same answer as the other _ARG macro, thanks!

Reply:
done, thank you!